### DISCUSSION TITLE
Custom icons in the Navigation items

### DISCUSSION THREAD

**Discussion by MinSomai** [payloadcms/payload#6768] [Feature Requests & Ideas]
Would love support for custom icons in the navigation menu. Maybe custom component to be passed in the collections `admin` field.

<img width="260" alt="Screenshot 2024-06-13 at 10 52 07 PM" src="https://github.com/payloadcms/payload/assets/21338691/80e9f2f7-cd65-4a64-8578-a5a0c9021a23">

Here is a wordpress one.

<img width="214" alt="Screenshot 2024-06-13 at 10 56 46 PM" src="https://github.com/payloadcms/payload/assets/21338691/6343ad2d-f12e-4acd-bda6-bb9ee8e02544">

I recently create a new item, forgot this already exists. New duplicate -> https://github.com/payloadcms/payload/discussions/9339

**Comment by auduongtuan** [comment]
Any updates on this?

**Comment by a-stein** [comment]
My workaround was to add emojis to the collections `label`. It would still be nice though to have the ability to add custom icons

**Comment by MinSomai** [comment]
this repo has example icons

https://github.com/akhrarovsaid/payload-theme-quantum-leap

**Reply by MinSomai** [reply]
this is interesting too, with plugin

https://github.com/tatsuyakari1203/payload-sidebar
